plugins/sentinelone/unit_test/test_get_events.py

```diff
@@ -60,7 +60,8 @@ def test_get_events(self, mock_request, test_name, input_params, expected):
             ],
         ]
     )
-    def test_get_events_raise_exception(self, mock_request, test_name, input_params, cause, assistance):
+    @patch("time.sleep", lambda _: None)
```

Comment:
I've misinformed you earlier. We're using freezegun to stick time to run the tests, and we're using mock sleep to prevent the backoff functionality in requests retries from delaying unit tests excessively.